assets/containerisableComponents/contributionAwarePaymentLogos/contributionAwarePaymentLogos.scss

```diff
@@ -0,0 +1,24 @@
+.contribution-aware-payment-logos {
+  position: absolute;
+
+  left: -197px;
+  top: 86%;
+
+  height: 20px;
+  svg {
+    height: 27px;
+    width: 42px;
+    margin-left: 4px;
+  }
+}
+
+.contribution-aware-payment-logos--one-off {
+  top: 70%;
+  left: -150px;
+
+  .svg-paypal-logo {
+    position: relative;
+    top: -56px;
+    left: -46px;
+  }
+}
```

Comment:
As discussed offline, I would be inclined to put the layout-related CSS (i.e. `position`, `top`, `left`) in each place that the logos are used, as it makes this component more reusable. However, if you prefer this I don't feel too strongly 🙂.

Reply:
Since the two only places that we are using this component have the same layout, I would prefer to keep it here and avoid duplication of code.